packages/webview_flutter/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -134,6 +133,68 @@ public void onProgressChanged(WebView view, int progress) {
     }
   }
 
+  /**
+   * Creates a {@link android.webkit.WebView} and configures it according to the supplied
+   * parameters.
+   *
+   * <p>The {@link WebView} is configured with the following predefined settings:
+   *
+   * <ul>
+   *   <li>always enable the DOM storage API;
+   *   <li>always allow JavaScript to automatically open windows;
+   *   <li>always allow support for multiple windows;
+   *   <li>always use the {@link FlutterWebChromeClient} as web Chrome client.
+   * </ul>
+   *
+   * <p><strong>Important:</strong> This method is visible for testing purposes only and should
+   * never be called from outside this class.
+   *
+   * @param webViewBuilder a {@link WebViewBuilder} which is responsible for building the {@link
+   *     WebView}.
+   * @param params creation parameters received over the method channel.
+   * @param webChromeClient an implementation of WebChromeClient This value may be null.
+   * @return The new {@link android.webkit.WebView} object.
+   */
+  @VisibleForTesting
+  static WebView createWebView(
+      WebViewBuilder webViewBuilder, Map<String, Object> params, WebChromeClient webChromeClient) {
+    Boolean usesHybridComposition = (Boolean) params.get("usesHybridComposition");
```

Comment:
Not directly, since the `params` is of type `Map<String, Object>` so the value could be `null` (requiring the `Boolean` reference type).

But I agree that this should be checked and if `null` return a default value of `false`. I will combine this with the map method discussed in the comment above.

Reply:
Since I decided not to go with the implementation of a `map()` method (see [this comment](https://github.com/flutter/plugins/pull/4178#discussion_r674647775)) I have handled this here correctly.